Cargo.toml

```diff
@@ -38,7 +38,7 @@ thiserror = "1.0"
 base64 = "0"
 rand = "0.8"
 prost = "0"
-daemonize = "0.4"
+daemonize = {git = "https://github.com/knsd/daemonize", rev = "f7be28e"}
```

Comment:
I didn't think we'd need a specific rev?

Reply:
We need something newer than the latest release which was tagged over a year ago. I was debating using the main branch but thought that opened things up for  unexpected changes.